src/Application.hs

```diff
@@ -12,7 +12,8 @@ module Application
     , handler
     ) where
 
-import Import
+-- TODO: How is threadDelay even exported from there? Should that be changed instead of hiding it here?
+import Import hiding (threadDelay)
```

Comment:
Ah, I see. Is the version from `Import` a generalised version of the one from `Control.Concurrent` which could also work in this module? If so, then we could just remove `threadDelay` from the `Control.Concurrent` import list. If not, I think the code you've got here is fine.

I would guess the one from Import is coming via Import.NoFoundation and from there, via ClassyPrelude.Yesod: https://github.com/purescript/pursuit/blob/e1dc3550ff88473bdfba003820743ddac0bcf688/src/Import/NoFoundation.hs#L6